great_expectations/datasource/data_connector/__init__.py

```diff
@@ -15,3 +15,7 @@
 from .inferred_asset_s3_data_connector import InferredAssetS3DataConnector
 from .inferred_asset_sql_data_connector import InferredAssetSqlDataConnector
 from .runtime_data_connector import RuntimeDataConnector
+
+from .configured_asset_azure_data_connector import (  # isort:skip
+    ConfiguredAssetAzureDataConnector,
+)
```

Comment:
style question: is there any reason we've got the mix of single- and multi-line imports in this file?

Reply:
I believe it is simply due to `black` formatting lines that pass a certain length.

`isort` moves this to the very top and causes a whole bunch of stuff to break down. I'm not too well versed in package management so I just through the `# isort:skip` tag on there. Let me know if you think we should change any of this!

Reply:
Gotcha, max line length setting is probably it; I'm not familiar with `black` and don't think this needs to be blocking for the moment, but I'll take a look at this when I'm touching that file again for the inferred connector.